### ISSUE TITLE
set the journal-entry timestamp with the API

### ISSUE THREAD

**Issue by brammeleman** [netbox-community/netbox#9648]
### NetBox version

v3.2.5

### Feature type

Change to existing functionality

### Proposed functionality

Set the date/time-stamp for new journal entries with the API.

### Use case

For migrating log/journal entries from another DCIM.

### Database changes

No changes

**Comment by jeremystretch** [comment]
> Set the date/time-stamp for new journal entries with the API.

This does not seem like something we want to expose in the REST API, as it opens the door for users to fabricate the creation times of journal entries.

**Comment by brammeleman** [comment]
I understand your concern.
Personally, as I just completed the migration from our old dcim to Netbox, I don't need this feature anymore. Except for the journal, I was able to do a pure API based import of all data. The journal entries (~2k) where inserted directly into the database, which is just a bit more work/complicated.

pros:
- beneficial for new Netbox users migrating from another dcim (no need to write to the database directly)

cons:
- special (admin) permission needed to back date and impersonate journal entry creation
- maybe too much work for a one-time import scenario
